ui/core.js

```diff
@@ -63,6 +63,13 @@ $.extend( $.ui, {
 			activeElement = document.body;
 		}
 
+		// Support: IE 9 - 11 only
+		// IE may return null instead of an element
+		// Interestingly, this only seems to occur when NOT in an iframe
+		if ( !activeElement ) {
```

Comment:
I prefer the clarity of what work arounds exist over the minuscule byte savings of combining.